content/_index.md

```diff
@@ -9,14 +9,14 @@ class: "no-pagination no-top-border-header no-search max-text-width"
 
 ##### What is OpenCensus?
 
-{{<title>}} is a vendor-agnostic single distribution of libraries to provide **metrics** collection and **tracing** for your services.
+{{<title>}} is a single distribution of libraries that automatically collect **metrics** and **distributed traces** from your services
```

Comment:
Interesting wording, I don't understand how we "automatically collect metrics and distributed traces"

Reply:
How about:

```suggestion
{{<title>}} is a single distribution of libraries that collect **metrics** and **distributed traces** from your services
```

Reply:
Sure, or even `to collect` as `that collect` makes it sound like the libraries passively do this.

Reply:
This was marked as resolved but unfortunately never changed